airflow-core/src/airflow/config_templates/config.yml

```diff
@@ -1428,6 +1428,42 @@ workers:
       sensitive: true
       example: ~
       default: ""
+    task_instance_min_heartbeat_interval:
```

Comment:
Is the `task_instance` prefix here redundant maybe? Since everything here is to do with workers.

Reply:
Alright yeah. I gave it a brief thought but thought to be explicit